cmd/operator-sdk/add/crd.go

```diff
@@ -29,11 +29,14 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const deprecationTemplate = "\033[1;36m%s\033[0m"
+
 // newAddCRDCmd - add crd command
 func newAddCRDCmd() *cobra.Command {
 	crdCmd := &cobra.Command{
-		Use:   "crd",
-		Short: "Adds a Custom Resource Definition (CRD) and the Custom Resource (CR) files",
+		Hidden: true,
```

Comment:
There's a convenient `Deprecated` field that you can use with cobra commands:
```go
type Command struct {
    ...
    // Deprecated defines, if this command is deprecated and should print this string when used.
    Deprecated string
    ...
}
```
When run, cobra will append a "this is deprecated," prefix to the string you set here, so you can set
```go
cmd.Deprecated = `Use 'operator-sdk add api' command to create a new API definition for a
custom resource.

'add api' will generate or update CRDs for a particular GVK based on values provided to
'--kind' and '--api-version' specified in the command.

Refer to 'https://sdk.operatorframework.io/docs/cli/operator-sdk_add_api/' for more details.`
```

Reply:
You can also remove hidden I believe, since the deprecated field will also hide the command.
.